addons/controllers/cpi/vspherecpiconfig_controller.go

```diff
@@ -139,6 +140,21 @@ func (r *VSphereCPIConfigReconciler) reconcileVSphereCPIConfigNormal(ctx context
 		r.Log.Error(err, "Error creating or patching VSphereCPIConfig data values secret")
 		return err
 	}
+
+	// deploy the provider service account for paravirtual mode
+	if cpiConfig.Spec.VSphereCPI.Mode == VSphereCPIParavirtualMode {
+		r.Log.Info("Create or update provider serviceAccount for cluster")
+
+		serviceAccount := &capvvmwarev1beta1.ProviderServiceAccount{}
+		_, err := controllerutil.CreateOrUpdate(ctx, r.Client, serviceAccount, func() error {
+			serviceAccount = r.mapCPIConfigToProviderServiceAccount(ctx, cpiConfig, cluster)
+			return controllerutil.SetControllerReference(cluster, serviceAccount, r.Scheme)
+		})
+		if err != nil {
+			r.Log.Error(err, "Error creating or updating ProviderServiceAccount for cluster")
```

Comment:
let's change the log to `for CPI`.
This psa is for CPI specifically

Reply:
fixed